.github/workflows/dapr-test.yml

```diff
@@ -604,6 +593,19 @@ jobs:
           message: |
             ## ✅ Tests succeeded on ${{ matrix.target_os }}/${{ matrix.target_arch }}
 
+            - Image tag: `${{ env.DAPR_TAG }}`
+            - Test image tag: `${{ env.DAPR_TEST_TAG }}`
+      - name: Update PR comment for skipped test run
+        if: success() && env.PR_NUMBER != '' && env.SKIP_E2E = 'true'
+        uses: marocchino/sticky-pull-request-comment@v2.2.0
+        with:
+          header: ${{ github.run_id }}
+          number: ${{ env.PR_NUMBER }}
+          append: true
+          GITHUB_TOKEN: ${{ secrets.DAPR_BOT_TOKEN }}
+          message: |
+            ## 😕 Tests skipped on ${{ matrix.target_os }}/${{ matrix.target_arch }}
```

Comment:
Should the emoji here be a yello ⚠️ (to be in line with the ❌ and ✅) ?

Reply:
Sure.